midp/localmsg.js

```diff
@@ -18,6 +18,15 @@ var LocalMsgConnection = function() {
     this.clientMessages = [];
 }
 
+LocalMsgConnection.prototype.reset = function() {
```

Comment:
nit: Consider implementing `LocalMsgConnection` (the constructor) in terms of `LocalMsgConnection.prototype.reset`. That way when we modify this class we only need to worry about initialization/reset in one place. I think the body of `LocalMsgConnection` could just be `this.reset()` if the `reset` function is modified slightly.